app/controllers/api/v3/medical_histories_controller.rb

```diff
@@ -45,6 +45,8 @@ def medical_histories_params
         :diabetes,
         :hypertension,
         :diagnosed_with_hypertension,
+        :bmi,
+        :is_smoking,
```

Comment:
Should this API be versioned? We want to keep the API backwards compatible with the client

Reply:
Generally, we want to be safer when making API changes because we can't control how app updates are rolled out to users (Android).

Reply:
It boils down to the client supporting it. We should capture this in the tech spec and have the Android devs approve this. We can also test this change locally with an un-updated Android version to verify that there aren't any issues. You can read more on the setup here: https://github.com/simpledotorg/simple-server?tab=readme-ov-file#developing-with-the-android-app